src/features/components/templates/footer/social-icons/index.tsx

```diff
@@ -9,14 +9,24 @@ import useVisibleContent from 'components/hooks/use-visible-content'
 import useRegion from 'components/hooks/use-region'
 
 const FooterSocialIcons = () => {
-    const current_path = getLocationPathname()
-    const splitted_path = current_path.split('/')
-    const is_career_page = splitted_path.includes('careers')
+    const [isCareer, setIsCareer] = useState(false)
```

Comment:
```suggestion
    const [is_career, setIsCareer] = useState(false)
```

Reply:
not sure are we using snake case for state names?

Reply:
You are correct 👍